civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -110,9 +110,9 @@
 
 // Imports from other styles
 // --------------------------------------------------
+@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/mixins/prefix';
```

Comment:
we already have a
```scss
@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/mixins/utility';
```
at the top of the file, either you make that line generic (import all the mixins), or put the two imports together

Reply:
SCSSROOT does not support wildcards so I have put both of them together.

Reply:
don't ask me why, but they decided to use "ALL" instead of "*", see [here](https://github.com/civicrm/civihr/blob/staging/org.civicrm.bootstrapcivihr/scss/civihr.scss#L2)
```scss
@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/mixins/ALL';
```